src/main/java/dto/Photo.java

```diff
@@ -7,5 +7,5 @@
     private int photoId;
```

Comment:
Code Review Input # 2 - Project Structure
I deleted several unnecessary folders from your project structure and moved the files to the appropriate places.
There were some nested folders that did not need to exist. (see all of the files above, where no changes were made to the code, but the route was changed)

According to Baeldung,  this general folder structure is consistent and standard:
src/main/java: For source files
src/main/resources: For resource files, like properties
src/test/java: For test source files
src/test/resources: For test resource files, like properties

Reference: https://www.baeldung.com/java-clean-code